backend/app/converters/lib/marcxml_base_map.rb

```diff
@@ -225,42 +223,6 @@ def corp_template
           :obj => :name_corporate_entity,
           :rel => :names,
           :map => name_corp_map,
-        },
-        "//datafield[@tag='410']" => {
-          :obj => :name_corporate_entity,
-          :rel => :names,
-          :map => name_corp_map,
-          :defaults => {
-            :name_order => 'direct',
-            :source => 'ingest'
-          }
-        },
```

Comment:
Perhaps this one should remain? `410: Series Statement/Added Entry-Corporate Name`.